java/org/apache/catalina/ha/tcp/SimpleTcpCluster.java

```diff
@@ -339,6 +339,13 @@ public void setChannelSendOptions(int channelSendOptions) {
         this.channelSendOptions = channelSendOptions;
     }
 
+    public void setChannelSendOptions(String channelSendOptions) {
+
+        int value = Channel.parseChannelSendOptions(channelSendOptions);
+        if (value > 0)
```

Comment:
Violeta, thank you for your feedback.

I'll add curly braces to the `for` block.

So each of those if's should have curly braces?

```if ("asynchronous".equals(opt) || "async".equals(opt))
            return SEND_OPTIONS_ASYNCHRONOUS;```

That's a lot of curly braces...  Perhaps a `switch` statement would be "cleaner" here then?